docs/tutorials/hyp3_insar_stack_for_ts_analysis.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    date_range=('2019-06-10', '2019-07-21')
</pre><p>Similar for this -- I don't think we ever use date_range again.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ASFHyP3/hyp3-docs/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>